components/progressbar/stories/metertemplate.js

```diff
@@ -0,0 +1,55 @@
+import { html } from "lit";
+import { classMap } from "lit/directives/class-map.js";
+import { Template as FieldLabel } from "@spectrum-css/fieldlabel/stories/template.js";
+
+import "../index.css";
+
+export const Template = ({
+	rootClass = "spectrum-ProgressBar",
+	customClasses = [],
+	label,
+	value,
+	meterFill,
+	size = "s",
+	...globals
+}) => {
+	const fillColor =
+		meterFill === "negative" ? "is-negative" :
+		meterFill === "positive" ? "is-positive" :
+		meterFill === "notice" ? "is-notice" : "default";
+
+	return html`
+		<div
+			class=${classMap({
+				[rootClass]: true,
+				["spectrum-Meter"]: true,
+				[`spectrum-Meter--size${size?.toUpperCase()}`]: size,
+				[`${fillColor}`]: fillColor !== "default",
```

Comment:
Rather than doing all of the conditionals above for this, could we just do something along the lines of 
```
[`is-${meterFill}`]: meterFill !== default
```

Reply:
Are you suggesting instead of  `const fillColor`? I think I do need that, it is adding the `is-` to the fill class.

Reply:
oh! Nice call, I like this a lot better.